dags/veda_data_pipeline/groups/transfer_group.py

```diff
@@ -51,6 +51,8 @@ def transfer_data(payload={}):
     airflow_vars_json = json.loads(airflow_vars)
     external_role_arn = airflow_vars_json.get("ASSUME_ROLE_WRITE_ARN")
     # (event, chunk_size=2800, role_arn=None, bucket_output=None):
+    if payload == {}:
+        payload = ti.dag_run.conf
```

Comment:
This kind of feels like a code smell 🤧 I would love feedback on this if it is!

Reply:
Yeah the payload vs run configuration management feels odd but this change looks like it will fix the standalone transfer for now without causing problems for the in-promotion-pipeline task.